pkg/sdk/testint/helpers_test.go

```diff
@@ -689,6 +689,7 @@ type informationSchemaColumns struct {
 	IdentityMinimum        sql.NullString `db:"IDENTITY_MINIMUM"`
 	IdentityCycle          sql.NullString `db:"IDENTITY_CYCLE"`
 	IdentityOrdered        sql.NullString `db:"IDENTITY_ORDERED"`
+	SchemaEvolutionRecord  sql.NullString `db:"SCHEMA_EVOLUTION_RECORD"`
```

Comment:
do I see correctly that the name differs than in table? (there are spaces and here underscores) Is it a difference in Snowflake or an error?

Reply:
This is what I see in `information_schema.columns`. Tests are passing, so I guess it's fine.